### PR TITLE
Fix #1669 on develop

### LINKED ISSUE #1669: service restart packager.io
HI,

Is it possible to do an automatic service restart at the end of the update on packager.io?

Have a nice day

### PR DESCRIPTION
**Pull request by preovaleo**
Fix #1669 in the develop branch
Restart the gogs service at the end of an update

### CODE REVIEW

**Comment by Unknwon** [comment]
Thanks again!

Based on https://github.com/gogits/gogs/issues/1669#issuecomment-142007106

@crohr suggests add this line to end of file?

File: packager/debian/postinst [hunk 1]
```diff
@@ -30,4 +30,6 @@ case "$1" in
     ${CLI} scale web=1 || true
   ;;
 
+service gogs restart || true
+
```
Comment (TheHowl) [inline packager/debian/postinst]: I'm no bash expert, but why did you write this line out of all the other cases? I mean, shouldn't it go before the `;;` at line 31?

**Comment by preovaleo** [comment]
Yes you're right solved

**Comment by Unknwon** [comment]
cc @TheHowl @crohr please help review!

File: packager/debian/postinst [hunk 2]
```diff
@@ -28,6 +28,12 @@ case "$1" in
 
     # scale
     ${CLI} scale web=1 || true
+    
+    #restart the service
```
Comment (TheHowl) [inline packager/debian/postinst]: I don't want to be picky here, but all the other comments are in the format `# <content>`. Why make this one `#<content>`?

File: packager/debian/postinst [hunk 3]
```diff
@@ -28,6 +28,12 @@ case "$1" in
...
+    service gogs restart || true
+    
   ;;
 
+
+
```
Comment (TheHowl) [inline packager/debian/postinst]: And also, why unnecessary newlines?

**Comment by Unknwon** [comment]
Thanks!

**Pull request merged**